networking/ingress-operator.adoc

```diff
@@ -47,6 +47,20 @@ include::modules/nw-ingress-setting-a-custom-default-certificate.adoc[leveloffse
 
 include::modules/nw-ingress-custom-default-certificate-remove.adoc[leveloffset=+2]
 
+include::modules/nw-autoscaling-ingress-controller.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional Resources
+* xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects[Enabling monitoring for user-defined projects]
+
+* xref:../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-install[Installing the custom metrics autoscaler]
+
+* xref:../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-trigger-auth[Understanding custom metrics autoscaler trigger authentications]
+
+* xref:../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-prometheus[Configuring the custom metrics autoscaler to use {product-title} monitoring]
+
+* xref:../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-adding[Understanding how to add custom metrics autoscalers]
```

Comment:
These last four bullets all dumped to the top of the assembly for me via the preview build. Maybe double-check the targets?

Reply:
Do you have a screenshot of this? I don't see this on my end

Reply:
ok, I see. Thanks for catching this! Fixed